docs/loading/Spark-connector-starrocks.md

```diff
@@ -348,3 +378,335 @@ The following example explains how to load data with Spark SQL by using the `INS
     +------+-----------+-------+
     2 rows in set (0.00 sec)
     ```
+
+## Best Practices
+
+### Load data to Primary Key table
+
+This section will show how to load data to StarRocks Primary Key table to achieve partial updates, and conditional updates.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/b2ee3973-fa30-4c13-a69b-3aa75988f331)
1. Add an article ("a").
2. Remove the comma.